features/articles/Article.js

```diff
@@ -128,7 +128,14 @@ const Article = ({
 
       <div className="wir-content-padding">
         <div className={styles['article-page-content']}>
-          {type === 'audio' && <AudioPlayer trackId={audio.id} />}
+          {type === 'audio' &&
+            (audio.id ? (
+              <AudioPlayer trackId={audio.id} />
+            ) : (
+              <div className={cn(b('content-unavailable'), typography['common-text'])}>
+                <Text id="article.podcast-will-be-available-soon" />
+              </div>
```

Comment:
maybe put it inside `AudioPlayer`?

Reply:
I don't know.. What would be more authentic in React?

Reply:
Done. Moved it into the player.